front/migrations/db/migration_132.sql

```diff
@@ -0,0 +1,2 @@
+-- Migration created on Dec 16, 2024
+ALTER TABLE "public"."agent_message_feedbacks" ADD COLUMN "conversationShared" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
mismatch of name of the column with the above?

Reply:
obviously 🤦